xAPI.md

```diff
@@ -1862,6 +1862,10 @@ The Experience API applies the concept of registration more broadly.  A registra
 considered to be an attempt, a session, or could span multiple Activities. There is no expectation that 
 completing an Activity ends a registration. Nor is a registration necessarily confined to a single Agent.
 
+The Registration id is also used when storing documents within the State resource, e.g. for 
```

Comment:
"Registration id" is a new concept; we should rephrase to avoid introducing it. Perhaps "A registration is also used when storing documents..." and "Normally, the same registration is used..."? We might even expand the second a little to something like "Activity Providers SHOULD use the same registration within Statements and when requesting State resources related to the same learning experience"?